src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs

```diff
@@ -1111,6 +1113,35 @@ public virtual async Task TestParallelDeleteMutations()
 
         }
 
+        /// <summary>
+        /// Performs create item on different Windows Regional format settings and validate that the data type Float is correct
+        /// </summary>
+        public async Task CanCreateItemWithCultureInvariant(string cultureInfo, string dbQuery)
+        {
+            CultureInfo ci = new(cultureInfo);
+            CultureInfo.DefaultThreadCurrentCulture = ci;
+
+            string graphQLMutationName = "createSales";
+            string graphQLMutation = @"
+                mutation {
+                    createSales (item: { item_name: ""test_name"", subtotal: 3.14, tax: 1.15 }) {
+                        id
+                        item_name
+                        subtotal
+                        tax
+                        total
+                    }
+                }
+            ";
+
+            JsonElement response = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            string dbResponse = await GetDatabaseResultAsync(dbQuery);
+            using JsonDocument dbResponseJson = JsonDocument.Parse(dbResponse);
+
+            // Validate results
+            Assert.AreEqual(Convert.ToDouble(dbResponseJson.RootElement.GetProperty("total").GetDouble(), CultureInfo.InvariantCulture), response.GetProperty("total").GetDouble());
```

Comment:
Why do you need to convert to double if you are already doing a GetDouble() operation?

Reply:
Convert.ToDouble() is used to change the double value based on different windows regional format settings. The GetDouble() operation is to change the JsonElement into an object that can be used by Convert.ToDouble().